src/views/product-root-docs-path-landing/index.tsx

```diff
@@ -30,42 +34,55 @@ const ProductRootDocsPathLanding = ({
 
 	let mdxSlot: ReactElement
 	if (mdxSource) {
-		const classes = classNames(s[`${product.slug}MDXWrapper`], s.mdxSlotWrapper)
+		const docsMdxComponents = getDocsMdxComponents(product.slug)
+		const classes = classNames(
+			s[`${product.slug}MDXWrapper`],
+			s.mdxSlotWrapper,
+			docsViewStyles.mdxContent
```

Comment:
Where was this class before?

Reply:
It was in `docs-view.module.css` before as well - but it was used indirectly because the entire `DocsView` component was rendered here within the `mdxSlot`. I think it could make sense to move the associated styles (prevent `iframe` overflow; remove margin on first item) into `DevDotContent`... but that would affect a bunch of places in the app so I was a little hesitant to start down that rabbit hole 😅 

Made a backlog task with some details at 🎟 [[maintenance] move docsViewStyles.mdxContent onto DevDotContent](https://app.asana.com/0/1202097197789424/1204107433878314/f)